src/test/java/org/jenkinsci/plugins/sonargerrit/inspection/ReportDataChecker.java

```diff
@@ -117,56 +113,56 @@ private static void checkReport1File(Report rep) {
         Assert.assertEquals("juice-events", c2.getPath());
 
         Component c3 = rep.getComponents().get(2);
-        Assert.assertEquals("com.megaproject.juice:juice-bootstrap", c2.getKey());
-        Assert.assertEquals("juice-bootstrap", c2.getPath());
+        Assert.assertEquals("com.megaproject.juice:juice-bootstrap", c3.getKey());
```

Comment:
Are these tests executed at all? All the tests passed even without this change, which is really strange...

Reply:
Method checkReport1File is not executed... I'll look into it, thanks.